packages/common/src/utils/dbt.ts

```diff
@@ -0,0 +1,3 @@
+export const validateDbtSelector = (selector: string): boolean =>
+    // eslint-disable-next-line no-useless-escape
+    /^@?[a-zA-Z0-9\*\-\.\+:_\/]+$/.test(selector);
```

Comment:
I added `/` and removed space (`\s`), none of the options need space:

https://docs.getdbt.com/reference/node-selection/syntax#shorthand

Reply:
White spaces are used to add multiple selectors: eg: `-s orders customers`